Automattic/woocommerce-payments
# Add internationalization to Credit Card payment method title for UPE

Fixes #3299 

#### Changes proposed in this Pull Request

Adds internationalization to Credit Card payment method title

I understand the need to not internationalize other payment methods as they're brand names/trademarks (e.g. giropfay, Sofort) but the default credit card payment cane be internationalized.

#### Testing instructions

1. Default installation of WooCommerce with WooCommerce Payments
2. Set default language to any language other than English
3. Add products to cart and access checkout
4. Verify that the Payment Method Title for credit cart payment has been localized

Before fix:
![01-WCP-before_fix](https://user-images.githubusercontent.com/1241486/142856879-4399300d-9dd6-478f-a9a9-95e902f3ca57.png)

After fix:
![02-WCP-after_fix](https://user-images.githubusercontent.com/1241486/142856903-681ea166-32ef-4b11-b5d0-642afedb1a51.png)

*

-------------------
